CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.0.0] - 2019-04-01
+### Changed
+- Published: When using Sequel all events checks `if` and `unless` predicates.
```

Comment:
*Sequel publishing hooks*: checking for `:destroy` events inside `:if`/`:unless` predicates